File: zkext/trunk/src/web/js/zkext/ui/form/FieldContainer.js
/* FieldContainer.js*/

zkext.ui.form.FieldContainer = zk.$extends(zkext.ui.container.Container,{	 
	$define: {	 
		 label:function(val){
			 this.setProperty("fieldLabel","setLabel",val);
		 }
	},
	configure_:function(){	
		this.$supers('configure_',arguments);		
		var config = this.getInitialConfig();
		
		//this.getInitialConfig().suspendLayout = true;
	},
	createExt_:function(){
		
		//this.newInstance('Ext.form.CheckboxGroup');
		this.newInstance('Ext.form.FieldContainer');
	}
});

File: zkext/trunk/src/com/ff/ui/form/FieldContainer.java
package com.ff.ui.form;

import com.ff.annotation.Property;
import com.ff.ui.container.Container;


public class FieldContainer extends Container{
	@Property
	private String label;

	public String getLabel() {
		return label;
	}

	public void setLabel(String label) {
		this.label = label;
	}
	
	
}
